packages/geo/src/lib/feature/shared/feature.utils.ts

```diff
@@ -233,3 +240,54 @@ export function moveToFeatures(
 export function hideOlFeature(olFeature: OlFeature) {
   olFeature.setStyle(new olstyle.Style({}));
 }
+
+/**
+ * Trys bind store layer
```

Comment:
Change description to: Try to bind a layer to a store if none is bound already. If no layer is given, a simple layer will be created.